doc-content/enterprise-only/openshift/environment-authoring-single-modify-proc.adoc

```diff
@@ -3,6 +3,7 @@
 
 By default, the single authoring template uses the H2 database with permanent storage. If you prefer to create a MySQL or PostgreSQL pod or to use an external database server (outside the OpenShift project), you need to modify the template before deploying the environment.
```

Comment:
I would remove "you need to":
By default, the single authoring template uses the H2 database with permanent storage. If you prefer to create a MySQL or PostgreSQL pod or to use an external database server (outside the OpenShift project), modify the template before deploying the environment.